src/bitcoin/poller/looper.rs

```diff
@@ -196,7 +196,13 @@ fn add_txs_to_db(
     // Now retrieve txs.
     let txs: Vec<_> = txids
         .into_iter()
-        .map(|txid| bit.wallet_transaction(&txid).map(|(tx, _)| tx))
+        .filter_map(|txid| {
+            if db_conn.get_tx(&txid).is_none() {
+                Some(bit.wallet_transaction(&txid).map(|(tx, _)| tx))
+            } else {
+                None
+            }
+        })
```

Comment:
In order to avoid adding a new DB method for this, I'll just get the existing txids from the current DB coins. There might be additional entries in the transactions table that are not referenced by the coins table, but this shouldn't happen often.

Reply:
both sound good to me!

Reply:
I've added a `list_saved_txids()` database method now and use this as a filter.